libsplinter/src/admin/service/error.rs

```diff
@@ -298,13 +298,17 @@ impl std::fmt::Display for AdminConsensusManagerError {
 pub enum AdminError {
     ConsensusFailed(AdminConsensusManagerError),
     MessageTypeUnset,
+    #[cfg(not(feature = "circuit-disband"))]
```

Comment:
Why does this need to be guarded by a feature?  Seems handy to have when we introduce new messages in general

Reply:
I added this error to fix a compiler warning happening in the `handle_message` admin service's function for when the disband functionality isn't compiled in because there is a match statement based on the `AdminMessage`'s message type (which is totally handled when the `circuit-disband` feature is enabled) and the error is in a catch-all case at the end when that message type isn't being handled. Currently, that function has a match statement that handles all of the possible cases.